grails-app/assets/javascripts/meriplan.js

```diff
@@ -404,7 +405,18 @@ function MERIPlan(project, projectService, config) {
     self.removeAsset = function(asset) {
         self.meriPlan().assets.remove(asset);
     };
-
+    self.addControl = function () {
```

Comment:
I don't think that add/remove row makes sense for the control approach - it's basically a yes/no with reasons.

Reply:
I've noticed just now on the latest doc file that Blaz sent yesterday that she also updated this section, she removed the option to add rows.

Reply:
I've pushed the changes for this